ImportStatus API calls seems to fail some times

It's not always that I see this error, but in PR #586, a simple doc change, failed during the testing phase due to an, apparently, API call failure not properly handled by `lab`.
```
--- FAIL: Test_fork (49.23s)
    --- FAIL: Test_fork/do_fork_namespace_name (1.66s)
        fork_test.go:120: Forking zaquestion/fork_test project...
            --- FAIL: TestLab (1.54s)
            panic: runtime error: invalid memory address or nil pointer dereference [recovered]
            	panic: runtime error: invalid memory address or nil pointer dereference
            [signal SIGSEGV: segmentation violation code=0x1 addr=0x60 pc=0xaeb298]
            
            goroutine 18 [running]:
            testing.tRunner.func1.1(0xfe1e80, 0x191a2c0)
            	/home/travis/.gimme/versions/go1.15.8.linux.amd64/src/testing/testing.go:1072 +0x30d
            testing.tRunner.func1(0xc000414480)
            	/home/travis/.gimme/versions/go1.15.8.linux.amd64/src/testing/testing.go:1075 +0x41a
            panic(0xfe1e80, 0x191a2c0)
            	/home/travis/.gimme/versions/go1.15.8.linux.amd64/src/runtime/panic.go:969 +0x1b9
            github.com/zaquestion/lab/internal/gitlab.Fork(0xc00010ba1f, 0x14, 0xc000739320, 0xc000030100, 0x23, 0x0, 0x0, 0xc00010ba70)
            	/home/travis/gopath/src/github.com/zaquestion/lab/internal/gitlab/gitlab.go:278 +0x3d8
            github.com/zaquestion/lab/cmd.forkFromOrigin(0xc00010ba1f, 0x14)
            	/home/travis/gopath/src/github.com/zaquestion/lab/cmd/fork.go:103 +0x3e5
            github.com/zaquestion/lab/cmd.glob..func10(0x192aae0, 0xc0002f06c0, 0x0, 0x4)
            	/home/travis/gopath/src/github.com/zaquestion/lab/cmd/fork.go:72 +0x413
            github.com/spf13/cobra.(*Command).execute(0x192aae0, 0xc0002f0680, 0x4, 0x4, 0x192aae0, 0xc0002f0680)
            	/home/travis/gopath/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:854 +0x2c2
            github.com/spf13/cobra.(*Command).ExecuteC(0x1928b60, 0xc000032180, 0x6, 0x6)
            	/home/travis/gopath/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
            github.com/spf13/cobra.(*Command).Execute(...)
            	/home/travis/gopath/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
            github.com/zaquestion/lab/cmd.Execute()
            	/home/travis/gopath/src/github.com/zaquestion/lab/cmd/root.go:241 +0x1c8
            github.com/zaquestion/lab.main()
            	/home/travis/gopath/src/github.com/zaquestion/lab/main.go:28 +0x93
            github.com/zaquestion/lab.TestLab(0xc000414480)
            	/home/travis/gopath/src/github.com/zaquestion/lab/main_test.go:16 +0x137
            testing.tRunner(0xc000414480, 0x120be88)
            	/home/travis/.gimme/versions/go1.15.8.linux.amd64/src/testing/testing.go:1123 +0xef
            created by testing.(*T).Run
            	/home/travis/.gimme/versions/go1.15.8.linux.amd64/src/testing/testing.go:1168 +0x2b3
            
        fork_test.go:121: exit status 2
```

Unfortunately, I wasn't able to reproduce the issue, but from a quick investigation, I noticed that the check for a fork status is not being checked in case an error is returned.

https://github.com/zaquestion/lab/blob/50a41b1c2527a671d8a20be8f3c9542912313523/internal/gitlab/gitlab.go#L275

I'm going to submit a PR with error handling. If that doesn't solve the issue, will give a better idea of what's going on.

--- a/internal/gitlab/gitlab.go
+++ b/internal/gitlab/gitlab.go
@@ -273,19 +273,23 @@ func Fork(project string, opts *gitlab.ForkProjectOptions, useHTTP bool, wait bo
 	//   https://docs.gitlab.com/ce/api/projects.html#fork-project
 	//   https://docs.gitlab.com/ee/api/project_import_export.html#import-status
 	status, _, err := lab.ProjectImportExport.ImportStatus(fork.ID, nil)
-	if wait {
-		for {
-			if status.ImportStatus == "finished" {
-				break
+	if err != nil {
+		log.Printf("Impossible to get fork status: %s\n", err)
+	} else {
+		if wait {
+			for {
+				if status.ImportStatus == "finished" {
+					break
+				}
+				status, _, err = lab.ProjectImportExport.ImportStatus(fork.ID, nil)
+				if err != nil {
+					log.Fatal(err)
+				}
+				time.Sleep(2 * time.Second)
 			}
-			status, _, err = lab.ProjectImportExport.ImportStatus(fork.ID, nil)
-			if err != nil {
-				log.Fatal(err)
-			}
-			time.Sleep(2 * time.Second)
+		} else if status.ImportStatus != "finished" {
+			err = errors.New("not finished")
 		}
-	} else if status.ImportStatus != "finished" {
-		err = errors.New("not finished")
 	}
 
 	urlToRepo := fork.SSHURLToRepo

